package.json

```diff
@@ -40,7 +40,8 @@
     "bluebird": "^3.5.5",
     "lodash": "^4.17.5",
     "request": "^2.83.0",
-    "request-promise": "^4.2.2"
+    "request-promise": "^4.2.2",
+    "smartcar": "^9.5.0"
```

Comment:
We don't need to modify this file, this was needed because your server.js has it as require.

A solution is to update the server.js with this line
const smartcar = require('.'); 

And revert this file